src/lib/db/feature-toggle-store.ts

```diff
@@ -262,7 +262,26 @@ export default class FeatureToggleStore implements IFeatureToggleStore {
             .where({ name: data.name })
             .update(this.dtoToRow(project, data))
             .returning(FEATURE_COLUMNS);
-        return this.rowToFeature(row[0]);
+
+        const feature = this.rowToFeature(row[0]);
+        // if a feature toggle's type or createdAt has changed, update its potentially stale status
```

Comment:
Can createdAt change? Is that correct?

Reply:
You can set it manually (and in fact some of the tests rely on it). That said, I don't think we allow you to send it through from the API. Maybe.